### ISSUE TITLE
info verb in virtual env docker image returns null version

### ISSUE THREAD

**Issue by murali-shris** [atsign-foundation/at_server#1005]
### Describe the bug

info verb in virtual env docker image returns null version

### Steps to reproduce
```
* setup local virtual environment using virtualenv:dev_env image
* connect to any secondary from virtual environment and run the info verb
* @alice🛠@info
data:{"version":null,"uptimeAsWords":"1 minutes 34 seconds","features":[{"name":"noop:","status":"Beta","description":"The No-Op verb simply does nothing for the requested number of milliseconds. The requested number of milliseconds may not be greater than 5000. Upon completion, the noop verb sends 'ok' as a response to the client.","syntax":"^noop:(?<delayMillis>\\d+)$"},{"name":"info:","status":"Beta","description":"The Info verb returns some information about the server including uptime and some info about available features. ","syntax":"^info(:brief)?$"}]}
```

### Expected behavior

* info verb should the correct version

### Additional context
failure in github actions for one of the tests which uses server version
https://github.com/atsign-foundation/at_client_sdk/actions/runs/3335376232/jobs/5519283223

**Comment by murali-shris** [comment]
@cpswan @XavierChanth 
Please review this PR. 
https://github.com/atsign-foundation/at_server/pull/1006
* Root cause of the issue is missing pubspec.yaml in docker image/virtual env.  
* Logic to place pubspec.yaml into the base folder in virtual environment should be done in 
https://github.com/atsign-foundation/at_server/blob/trunk/.github/workflows/at_server.yaml?
* how can I verify the fix in the PR?

**Comment by cpswan** [comment]
Fixed by #1006 

```
@info
data:{"version":"3.0.25+gha12345","uptimeAsWords":"39 seconds","features":[{"name":"noop:","status":"Beta","description":"The No-Op verb simply does nothing for the requested number of milliseconds. The requested number of milliseconds may not be greater than 5000. Upon completion, the noop verb sends 'ok' as a response to the client.","syntax":"^noop:(?<delayMillis>\\d+)$"},{"name":"info:","status":"Beta","description":"The Info verb returns some information about the server including uptime and some info about available features. ","syntax":"^info(:brief)?$"}]}
```

**Comment by murali-shris** [comment]
> Fixed by #1006
> 
> ```
> @info
> data:{"version":"3.0.25+gha12345","uptimeAsWords":"39 seconds","features":[{"name":"noop:","status":"Beta","description":"The No-Op verb simply does nothing for the requested number of milliseconds. The requested number of milliseconds may not be greater than 5000. Upon completion, the noop verb sends 'ok' as a response to the client.","syntax":"^noop:(?<delayMillis>\\d+)$"},{"name":"info:","status":"Beta","description":"The Info verb returns some information about the server including uptime and some info about available features. ","syntax":"^info(:brief)?$"}]}
> ```

server functional tests were still returning null. Worked after i made the below change
https://github.com/atsign-foundation/at_server/pull/1008
